### PR TITLE
Feature/788 community track venue gallery

### LINKED ISSUE #788: [Feature Request] Community Track Venue Gallery
## Is your feature request related to a problem?
Nope.

## Describe the solution you'd like
A clear information page that shows the photos of all the venues for the community track.

## Additional context
This page is mainly for the sponsor team to communicate with potential sponsors.

## Invite the community to discuss and vote
Earn votes and support by involving the community and invite everybody to discuss your feature and
to vote for it (votes are +1 , 0 or -1 via adding a **thumbs-up/down**)

### PR DESCRIPTION
**Pull request by freddy50806**
(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ * ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
This pr is for issue #788 

## Steps to Test This Pull Request
1. Go to '/en-us/sponsor/community-tracks/' and '/zh-hant/sponsor/community-tracks/'
2. Click on gallery and check pic is right

## Expected behavior
Every gallery is work and picture is right

## Related Issue
close #788 

## More Information

vueue-hant

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: src/templates/pycontw-2020/contents/en/sponsor/community-tracks.html [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+{% endblock styles %}
+{% endcompress %}
+{% comment %} {% block extrastyle %}
+
+{% endblock %} {% endcomment %}
+
+
+{% block content %}
+<section class="pill-title">
+    <h1>Conference Venue</h1>
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/en/sponsor/community-tracks.html:32]: 這邊叫做 `Community Track Venue` 好了。

File: src/templates/pycontw-2020/contents/zh/sponsor/community-tracks.html [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+{% endblock styles %}
+{% endcompress %}
+{% comment %} {% block extrastyle %}
+
+{% endblock %} {% endcomment %}
+
+
+{% block content %}
+<section class="pill-title">
+    <h1>Venue</h1>
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/zh/sponsor/community-tracks.html:32]: 這邊叫做 `社群軌場地` 好了。

Comment (freddy50806) [reply src/templates/pycontw-2020/contents/zh/sponsor/community-tracks.html]: add in commit 60d3ba1

**Comment by freddy50806** [comment]
所有建議都在 60d3ba1 修正

File: src/templates/pycontw-2020/contents/en/sponsor/prospectus.html [hunk 3]
```diff
@@ -307,7 +307,7 @@ <h1>Sponsorship Prospectus</h1>
...
     </li>
     <li>
-        PyCon TW 2020 community track is currently divided into 5 sectors. The PyCon TW organization holds the final right to decide the location of each sponsor. Diamond and Platinum sponsors have the highest priority for venue selection (already included in the sponsorship plan). Gold sponsors have the option to select subscriptions (right after Diamond and Platinum). The total amount is limited, so the first-come-first-served is adopted. Priorities can pre-subscribe and choose venue.
+        PyCon TW 2020 community track is currently divided into 5 sectors. The PyCon TW organization holds the final right to decide the location of each sponsor. Diamond and Platinum sponsors have the highest priority for venue selection (already included in the sponsorship plan). Gold sponsors have the option to select subscriptions (right after Diamond and Platinum). The total amount is limited, so the first-come-first-served is adopted. Priorities can pre-subscribe and <a href="/sponsor/community-tracks/" target="_blank">choose venue</>.
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/en/sponsor/prospectus.html:310]: oh shoot 我整個漏看，@freddy50806 這邊 link 不能直接這樣加 hard link，請用
```html
{% url 'page' path='sponsor/community-tracks' as community_tracks_url %}

...

<a href="{{ community_tracks_url }}">...</a>
```

staging & production 會自動在前面加上年份，但是 development 環境不會，所以開發環境會看不出來，部署後會有問題。我先 revert 了，再麻煩修正，感激。下面中文版的亦然。

**Pull request merged**